### ISSUE TITLE
Without pretraining on first 50 classes

### ISSUE THREAD

**Issue by ashok-arjun** [arthurdouillard/incremental_learning.pytorch#50]
Hi @arthurdouillard ,

Thanks for the great codebase!

I would like to evaluate all algorithms in the setting where pretraining on the first 50 classes is not allowed.
I'd like to know if just setting `--initial-increment` and `--increment` to appropriate values (eg. 5 and 5) would do.

Would it need any change in hyperparameters, in any of the algorithms?

Thank you very much!

**Comment by arthurdouillard** [comment]
> Thanks for the great codebase!

Thank you! 😊 

>  I'd like to know if just setting --initial-increment and --increment to appropriate values (eg. 5 and 5) would do.

Yes it'll work perfectly, just so you know when initial increment = increment, the community often also disabled the fixed memory (see options file).

> Would it need any change in hyperparameters, in any of the algorithms?

I'm sure you could improve PODNet or UCIR bu tuning them yes, but neither them or I did. Note that those algos are a bit particular, and are largely SotA when initial increment is large, but their gains is less important with small increments. I think it's because they are metric-based, with a cosine classifier, and thus need more pretraining data.

**Comment by ashok-arjun** [comment]
Got it, Arthur. Thank you very much for the reply.
